platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -29,6 +29,7 @@
 MLNX_NUM_PSU = 2
 
 GET_HWSKU_CMD = "sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku"
```

Comment:
is this still needed and used?

Reply:
This is still required to set the name of the chassis, as it is done in the below code:
        mi = get_machine_info()
        if mi is not None:
            self.name = mi['onie_platform']
        else:
            self.name = self.sku_name